src/components/v5/shared/MenuWithStatusText/MenuWithStatusText.tsx

```diff
@@ -19,7 +22,9 @@ const MenuWithStatusText: React.FC<MenuWithStatusTextProps> = ({
       className: 'bg-gray-50 !py-3',
       content: (
         <>
-          <StatusText {...restStatusTextSectionProps} />
+          <LoadingSkeleton isLoading={isLoading} className="h-4 w-full rounded">
```

Comment:
This could end up quite some work as I see this component being used in 14 places:
![image](https://github.com/user-attachments/assets/65b577f0-c281-4d77-ac3c-d269d8a90287)

I think it's about time we start passing in `statusText` as a rendered node and we just display it here. We can then pull the `content` property a level up, which would make us use this something along these lines:
![image](https://github.com/user-attachments/assets/ec5fc3f1-536a-4a67-8ff2-3876b7e18931)
and use the component like this (I used `VotingStep` as an example:
![image](https://github.com/user-attachments/assets/dd8e8526-09e6-435f-b1be-6d404e3a4175)

And then you could just pass what you have just wrote wherever appropriate and this component stops caring about loading states and just does layout :chart_with_upwards_trend:

Reply:
@bassgeta done. I dedicated to create a separate commit to change this component and adjust thait in other parts in the app